### ISSUE TITLE
JS assets are much larger on staging - issue with web pack?

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#1189]
**Staging assets:**
1	HTML/Text	11.6K
8	JavaScript File	719.2K
3	Stylesheet File	129.6K
4	CSS Image	46.6K
1	Favicon	3.2K

**Production assets (pre web pack)**
1	HTML/Text	5.5K
14	JavaScript File	258.2K
3	Stylesheet File	132.9K
4	CSS Image	46.5K
1	Favicon	3.2K

**Comment by awong-dev** [comment]
@dannychapman Thanks!  Can you give me a pointer to how you are getting these numbers? I want to be sure I'm seeing the same thing.

**Comment by dannychapman** [comment]
@awong-dev sure thing - I've been using the YSlow plugin for Chrome. https://chrome.google.com/webstore/detail/yslow/ninejjcohidippngpapiilnmkgllmakh

**Comment by awong-dev** [comment]
Thanks! Found the problem.

webpack is not minifying in staging because I had omitted the -p flag (https://webpack.github.io/docs/cli.html#production-shortcut-p) which implies both `--optimize-minimize` and `--optimize-occurence-order`. When I run this locally, the javascript size goes to 309.7k. Worse, but not crazy bad. Some of it might be ES6 polyfills...I'd need to do a deeper analysis.

Minifying about doubles the time needed to run webpack. I need to think through the build scripts a bit so that we can keep local development fast without making staging diverge from production in such a big way.
